.github/workflows/on_pr.yaml

```diff
@@ -13,7 +13,6 @@ jobs:
     - name: generate-tag
       uses: phil-inc/public-actions/.github/actions/generate-tag@master
     - name: build-push
-      needs: generate-tag
       if: needs.generate_tag.outputs.tag != ''
```

Comment:
will this `if` work?

Reply:
Sure won't! Good catch, this should be fixed now.